### PR TITLE
[junit5] fix test runtime stats

### PR DESCRIPTION
**Pull request by calcwu**
- expand test type
- reset `startTime` for each test. 

Before 
```
<testsuite failures="0" time="8.288" errors="0" skipped="0" tests="8" name="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest">
<testcase time="0.686" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingClassTestBaseIsNotAppliedToEnum"/>
<testcase time="0.74" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingClassTestBaseIsNotAppliedToInstantiableClass"/>
<testcase time="0.757" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingBestPracticeFollowedInStaticMethods"/>
<testcase time="0.762" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingBestPracticeFollowedInConstructors"/>
<testcase time="0.789" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingBestPracticeFollowedInInstanceMethods"/>
<testcase time="1.489" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="validateInsertDefaultAccountCreationRule"/>
<testcase time="1.529" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="insertRemovesDuplicates"/>
<testcase time="1.536" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="testRangeValidation"/>
</testsuite>
```

After - notice the time is no longer incremental. 
```
<testsuite failures="0" time="2.979" errors="0" skipped="0" tests="8" name="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest">
<testcase time="0.852" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingClassTestBaseIsNotAppliedToEnum"/>
<testcase time="0.11" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingClassTestBaseIsNotAppliedToInstantiableClass"/>
<testcase time="0.043" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingBestPracticeFollowedInStaticMethods"/>
<testcase time="0.019" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingBestPracticeFollowedInConstructors"/>
<testcase time="0.066" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="nullCheckingBestPracticeFollowedInInstanceMethods"/>
<testcase time="1.756" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="validateInsertDefaultAccountCreationRule"/>
<testcase time="0.122" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="insertRemovesDuplicates"/>
<testcase time="0.011" classname="com.addepar.domain.service.accountimportmapping.AccountCreationRuleServiceImplTest" name="testRangeValidation"/>
</testsuite>
```

@scottlee14 @csaade @ztai-add  @Addepar/amp-crew

### CODE REVIEW

File: src/com/facebook/buck/testrunner/Junit5TestListener.java [hunk 1]
```diff
@@ -179,7 +180,7 @@ public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult
 
     String className = testClass.getCanonicalName();
     String methodName = testIdentifier.getDisplayName().replace("()", "");
-    long runTime = summary.getTimeFinished() - summary.getTimeStarted();
```
Comment (csaade) [inline src/com/facebook/buck/testrunner/Junit5TestListener.java:182]: nit: Should we delete the `summary#getTimeFinished` and `summary#getTimeStarted` if they are now unused ?

**Review by csaade — APPROVED** [review]
LGTM - cool!

**Review by ztai-add — APPROVED** [review]

File: src/com/facebook/buck/testrunner/Junit5TestListener.java (hunk 1 above)
Comment (calcwu) [reply src/com/facebook/buck/testrunner/Junit5TestListener.java:182]: That's a junit5 class.

**Review by jieyulu-addepar — APPROVED** [review]
💯

**Review by scottlee14 — COMMENTED** [review]
LGTM

**Pull request merged**